raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -696,10 +719,12 @@ contract TokenNetwork is Utils {
         uint256 participant1_transferred_amount,
         uint256 participant1_locked_amount,
         bytes32 participant1_locksroot,
+        Claim memory participant1_claim,
         address participant2,
         uint256 participant2_transferred_amount,
         uint256 participant2_locked_amount,
-        bytes32 participant2_locksroot
+        bytes32 participant2_locksroot,
+        Claim memory participant2_claim
```

Comment:
Is it guaranteed (or rather do we assume) that claims will always be issued pair-wise?

Reply:
It is not guaranteed. Since I don't see a way to pass something like `None` instead of a solidity struct, my current approach would be to not check the signature when the amount is 0, so that the client can pass in a dummy claim instead. Better suggestions welcome!